public/app/features/plugins/admin/components/PluginDetailsRightPanel.tsx

```diff
@@ -0,0 +1,53 @@
+import { PageInfoItem } from '@grafana/runtime/src/components/PluginPage';
+import { TextLink, Stack, Text } from '@grafana/ui';
+import { Trans } from 'app/core/internationalization';
+import { formatDate } from 'app/core/internationalization/dates';
+
+import { CatalogPlugin } from '../types';
+
+type Props = {
+  info: PageInfoItem[];
+  plugin: CatalogPlugin;
+};
+
+export function PluginDetailsRightPanel(props: Props): React.ReactElement | null {
+  const { info, plugin } = props;
+
+  return (
+    <Stack direction="column" gap={2} grow={0} shrink={0}>
```

Comment:
I personally think `gap={1}` would do just fine - both here and below, but this is more of a design question. so treat this is a nit.